CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project aims to adhere to [Semantic Versioning](http://semver.org/).
 
 ## [3.4.1]
 ### Added
+ - [Fixed bug](https://github.com/joyent/java-manta/pull/527) where
```

Comment:
Apologies for being nitty here, but you could reference the issue fixed under 

```
###Fixed
[Specifying key content will conflict with the default key path](https://github.com/joyent/java-manta/issues/116)
```

Also, you could re-edit 
```
[Fixed Bug](https://github.com/joyent/java-manta/pull/527) -> [Updated](https://github.com/joyent/java-manta/pull/527)
```